src/Testcontainers.Ollama/OllamaBuilder.cs

```diff
@@ -0,0 +1,121 @@
+namespace Testcontainers.Ollama
+{
+    /// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+    [PublicAPI]
+    public sealed class OllamaBuilder : ContainerBuilder<OllamaBuilder, OllamaContainer, OllamaConfiguration>
+    {
+        /// <summary>
+        /// Gets the default port of the Ollama API.
+        /// </summary>
+        public const int DefaultPort = 11434;
+        
+        /// <summary>
+        /// Default image name and version tag.
+        /// </summary>
+        public const string OllamaImage = "ollama/ollama:0.1.22";
+    
+        /// <summary>
+        /// Default volume path.
+        /// </summary>
+        public const string DefaultVolumePath = "/root/.ollama";
```

Comment:
Volume mounting should be optional rather than the default, at least for now. We can align along with Java and Go implementation.